packages/SwingSet/tools/bootstrap-relay.js

```diff
@@ -76,51 +69,34 @@ export const buildRootObject = () => {
         };
         return getValue;
       };
-      const methodReturnValues = decodePassable(encodedMethodReturnValues);
       // `objectMap` hardens its result, but...
-      const methods = objectMap(methodReturnValues, makeGetterFunction);
+      const methods = objectMap(returnValues, makeGetterFunction);
       // ... `Far` requires its methods argument not to be hardened.
       const unhardenedMethods = { ...methods };
       const remotable = Far(label, unhardenedMethods);
```

Comment:
This could probably be made a little more concise.
```js
      // `Far` requires an unfrozen methods object,
      // so we shallow-copy the hardened result from `objectMap`.
      const methods = objectMap(returnValues, makeGetterFunction);
      const remotable = Far(label, { ...methods });
```

Reply:
Not my code, but that does seem like an improvment.